### DISCUSSION TITLE
a problem about apisix-java-plugin-runner

### DISCUSSION THREAD

**Discussion by foxk2fox** [apache/apisix#6272] [Q&A]
a ran apisix with apisix-java-plugin-runner ,and sent a request ,but I got this log in error.log ,'init.lua:703: phase_func(): failed to receive RPC_PREPARE_CONF: timeout','run_plugin(): ext-plugin-pre-req exits with http status code 503'....how to solve it

**Comment by tzssangglass** [comment]
Maybe you broke the debugging point, https://github.com/apache/apisix-java-plugin-runner/issues/77

and you can submit an issue to apisix-java-plugin-runner, give the complete reproduction steps.

**Reply by jiuchongxiao** [reply]
do you solved?

**Reply by jiuchongxiao** [reply]
how to solve,i have the same issue
